### PR TITLE
Restore binary test

### PR DESCRIPTION
**Pull request by vvscode**

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -20,6 +22,12 @@ const SEPARATOR = '\u001F';
 test.beforeEach(async () => pify(exec)(`rm -rf ${dest} ${destUnpacked}`));
 
 test('equals to sample', async t => {
+  // 1482680798652 - datetime for test/fixtures/output.apkg
+  const now = 1482680798652;
+  console.log('Now:', now);
```
Comment (ewnd9) [inline test/test.js]: Please remove the log statement

File: test/test.js [hunk 2]
```diff
@@ -20,6 +22,12 @@ const SEPARATOR = '\u001F';
 test.beforeEach(async () => pify(exec)(`rm -rf ${dest} ${destUnpacked}`));
 
 test('equals to sample', async t => {
+  // 1482680798652 - datetime for test/fixtures/output.apkg
```
Comment (ewnd9) [inline test/test.js]: Please remove the comment

Comment (vvscode) [inline test/test.js]: I'll remove. But after I'll got situation where I have ```const now = 1482680798652;``` with some magic-number without any comment about it's nature. 

Not cool for me

**Comment by vvscode** [comment]
https://github.com/ewnd9/anki-apkg-export/pull/15#discussion_r93845266

File: test/test.js (hunk 1 above)
Comment (vvscode) [inline test/test.js]: done

File: test/test.js (hunk 2 above)
Comment (ewnd9) [inline test/test.js]: Giving variable name `now` is self-explanatory. In the future, I recommend using comments like

```js
const now = 1482680798652; // Dec 25 2016, could be any date
```

**Pull request merged**